services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -220,33 +271,40 @@ async def add_project_node(
         request.app, user_id, project["uuid"]
     )
 
-    if not _is_node_dynamic(service_key):
-        return node_uuid
+    if _is_node_dynamic(service_key):
+        with suppress(ProjectStartsTooManyDynamicNodes):
+            # NOTE: we do not start the start if there are already too many
```

Comment:
shouldn't we inform the front-end from this? i.e. log message?

Reply:
no right now there is no option to do this. the node is added so it returns a 20X already. let's not start some weird stuff with 20X+message.
I think this should come later with the websocket communication.